src/components/dialogmote/innkalling/DialogmoteInnkallingBehandler.tsx

```diff
@@ -30,9 +31,9 @@ const DialogmoteInnkallingBehandler = ({
     <DialogmoteInnkallingSkjemaSeksjon>
       <BehandlerTittel>{texts.title}</BehandlerTittel>
       <BehandlerInfoRad>
-        {`${capitalizeFoersteBokstav(behandler.type.toLowerCase())}: ${
-          behandler.fornavn
-        } ${behandler.etternavn}`}
+        {`${capitalizeFoersteBokstav(
```

Comment:
Hva med å endre capitalizeFoersteBokstav i `src/utils/stringUtils.ts` til følgende?
```ts
export const capitalizeFoersteBokstav = (ord: string): string => {
  return ord.charAt(0).toUpperCase() + ord.slice(1).toLowerCase();
}
```
Det virker mer lesbart for min egen del

Reply:
Det kommer en ny metode for det i PRen fra June og meg 👍

Reply:
Usikker på om vi ønsker lowercase resten alle steder denne brukes, men kan lage en ny variant som gjør det 👍

Reply:
Ok, da lar jeg den være